package.json

```diff
@@ -4,15 +4,17 @@
   "workspaces": [
     "map",
     "site",
-    "web-client",
-    "api"
+    "web-client"
```

Comment:
you need to add the `functions` package here

Reply:
See an earlier comment regarding that :)